docs/source/index.rst

```diff
@@ -12,110 +12,15 @@ and visualizations. By encoding a complete and reproducible record of a
 computation, the documents can be shared with others on GitHub, Dropbox, and
 the `Jupyter Notebook Viewer <https://github.com/jupyter/nbviewer>`__.
 
-The following summary highlights the various open source projects in the
-Jupyter and IPython organizations.
+Project Jupyter consists of a large number of 
```

Comment:
Oops, stray sentence, pushing fix...

On Thu, Sep 3, 2015 at 12:06 PM, Matthias Bussonnier <
notifications@github.com> wrote:

> In docs/source/index.rst
> <https://github.com/jupyter/jupyter/pull/31#discussion_r38684074>:
>
> > @@ -12,110 +12,15 @@ and visualizations. By encoding a complete and reproducible record of a
> >  computation, the documents can be shared with others on GitHub, Dropbox, and
> >  the `Jupyter Notebook Viewer <https://github.com/jupyter/nbviewer>`__.
> >
> > -The following summary highlights the various open source projects in the
> > -Jupyter and IPython organizations.
> > +Project Jupyter consists of a large number of
>
> of what ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/jupyter/pull/31/files#r38684074>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com